cirq-core/cirq/protocols/act_on_protocol.py

```diff
@@ -86,9 +86,22 @@ def _act_on_(
         """
 
 
+def _fix_deprecated_args(args, kwargs):
+    kwargs['sim_state'] = kwargs['args']
+    del kwargs['args']
+    return args, kwargs
```

Comment:
[No change needed] If nothing else, the `args` / `'args'` confusion here is a good motivator for this change :smile: